tests/pyincore/analyses/mlenabledcgeslc/test_mlcgeslc.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from pyincore import IncoreClient, Dataset, globals as pyglobals
+from pyincore.analyses.mlenabledcgeslc import MlEnabledCgeSlc
+
+client = IncoreClient()
+
+mlcgeslc = MlEnabledCgeSlc(client)
+
+curr_dir = os.path.dirname(__file__)
```

Comment:
Before this is merged, we should push the sector_shocks data to incore-dev and read it as a dataset. We shouldn't push data into the source repository.

Reply:
Yes, I have addressed this.